### DISCUSSION TITLE
wake up

### DISCUSSION THREAD

**Discussion by tommo69** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1289] [General]
Just wondering if the app can also keep the Mac awake while connected in clamshell to an external display and unplugged (i.e. like Amphetamine)

**Comment by MatthiasKerbl** [comment]
Hi @tommo69,

In short, no. AlDente does only keep your mac awake in certain scenarios. All of them require the MacBook to be plugged in.
You can read more about this topic in the following blog post "[When Does AlDente Disable Sleep on MacBooks?](https://apphousekitchen.com/when-does-aldente-disable-sleep-on-macbooks/)".

Best,
Matthias
